content/docs/tasks/traffic-management/egress-gateway.md

```diff
@@ -51,7 +51,35 @@ If we used the [sleep](https://github.com/istio/istio/tree/{{<branch_name>}}/sam
 
 First let's direct HTTP traffic without TLS origination
 
-1.  Create an egress `Gateway` for _edition.cnn.com_, port 80:
+1.  Create an egress `Gateway` for _edition.cnn.com_, port 80.
+
+    If you have [mutual TLS Authentication](/docs/tasks/security/mutual-tls/) enabled in Istio, you must create the
+    `Gateway` in the following way:
```

Comment:
You should explain why you must create it this way, esp. why the host is "*" vs. "edition.cnn.com" in the non mTLS case.

Reply:
@frankbu I think I will just change the hosts to '*' in both cases since for HTTP it does not matter anyway. We may want to rethink this configuration later.